src/redis-check-rdb.c

```diff
@@ -80,6 +80,8 @@ char *rdb_type_string[] = {
     "stream-v2",
     "set-listpack",
     "stream-v3",
+    "hash-md-hashtable",
+    "hash-md-listpack",
```

Comment:
what about let md at the end, like `hash-hashtable-md`, if we add new ones in the future we won't continue these styles.